gix-revision/src/spec/parse/function.rs

```diff
@@ -422,7 +422,12 @@ where
                 if n < 0 {
                     if name.is_empty() {
                         delegate
-                            .nth_checked_out_branch(n.abs().try_into().expect("non-negative isize fits usize"))
+                            .nth_checked_out_branch(
```

Comment:
Isn't the only negative `isize` for there is no positive `isize` value `isize::MIN` which could be special cased to turn into `isize::MAX as usize + 1`? So no need to turn it into an error?

Reply:
Or if msrv is at least 1.51 there is also [isize::unsigned_abs](https://doc.rust-lang.org/std/primitive.isize.html#method.unsigned_abs)